org.osgi.test.cases.jdbc/src/org/osgi/test/cases/jdbc/junit/JDBCTestCase.java

```diff
@@ -77,20 +84,24 @@ private static class DfltProps extends Properties {
 	static String				description										= "desc";
 	static String				password										= "pswd";
 	static String				user											= "usr";
-
-	@Test
+	
+	@InjectBundleContext
+	BundleContext bundleContext;
+	
+	/**
+	 * Test that the JDBS service defines at least one capability
+	 * 
+	 * @param sr the service reference parameter
+	 */
+	@ParameterizedTest
+	@ServiceSource(serviceType = DataSourceFactory.class)
```

Comment:
Doesn't this PR contain changes which are the same as the other PR? We should merge the other PR first and then rebase this PR.

Reply:
Yep see description, the other should be merged first and I'll then rebase it.